specs/agents/tracing-instrumentation-aws-lambda.md

```diff
@@ -132,6 +132,66 @@ In version 2.0, the `${event.requestContext.routeKey}` can have the format `GET
 
 If `use_path_as_transaction_name` is applicable and set to `true`, use `${event.requestContext.http.method} ${event.requestContext.http.path}` as the transaction name.
 
+### Elastic Load Balancer (ELB)
+
+Elastic Load Balancer (ELB) can be attached directly to lambda, without the use
+of API Gateway. In this case the `event` object will be structured differently.
+
+The agent should use the information in the request and response objects to
```

Comment:
This is problem the best spec URL for the request and response payloads: https://docs.aws.amazon.com/elasticloadbalancing/latest/application/lambda-functions.html#receive-event-from-load-balancer (e.g. describing a possible `multiValueQueryStringParameters` field in the response).
Perhaps it is worth linking to that.

Reply:
Done